core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-action.service.ts

```diff
@@ -468,6 +470,21 @@ export class WorkflowActionService {
     this.executeAndStoreCommand(command);
   }
 
+  public addCommentBox(commentBox: CommentBox): void {
+    const currentHighlights = this.jointGraphWrapper.getCurrentHighlights();
+    const command: Command = {
+      modifiesWorkflow: true,
+      execute: () => {
+        this.jointGraphWrapper.unhighlightElements(currentHighlights);
+        this.jointGraphWrapper.setMultiSelectMode(false);
+        this.addCommentBoxInternal(commentBox);
+      },
+      undo: () => {
+        console.log("haha, just kidding");
```

Comment:
remove this line? also what should be done here for undo?

Reply:
Removed. Now undo() is deleting the commentBox with the corresponding CommentBoxID.